configs/eslint/node/typescript/index.js

```diff
@@ -53,7 +51,10 @@ module.exports = {
         browser: false,
         node: true,
       },
-      extends: ['plugin:n/recommended'],
+      extends: [
+        'plugin:@typescript-eslint/disable-type-checked',
```

Comment:
I disabled type-checking `.eslintrc.cjs` and `.prettierrc.cjs`, to work around a common issue reported in [I get errors telling me "ESLint was configured to run ... However, that TSConfig does not / none of those TSConfigs include this file"](https://typescript-eslint.io/linting/troubleshooting#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file).